meson.build

```diff
@@ -142,6 +140,46 @@ if git.found()
 endif
 add_project_arguments('-DSWAY_VERSION=@0@'.format(version), language: 'c')
 
+# Compute the relative path used by compiler invocations.
+source_root = meson.source_root().split('/')
```

Comment:
It would be better to use `meson.current_source_dir()` instead. It's unlikely sway would ever be used as a subproject, but I think it would return the directory of the parent project instead of sway's actual source root.

This would be more relevant for wlroots, though.

Reply:
Fixed; fortunately `meson.current_source_dir()` also returns an absolute path.